CMakeLists.txt

```diff
@@ -67,12 +67,16 @@ add_library(spvgen SHARED "")
 set_target_properties(spvgen PROPERTIES PREFIX "")
 
 if(EXISTS ${PROJECT_SOURCE_DIR}/../llpc/tool/vfx)
-    set(XGL_VFX_PATH ${PROJECT_SOURCE_DIR}/../llpc/tool/vfx CACHE PATH "The path of vfx.")
+    if ("${XGL_VFX_PATH}" STREQUAL "${PROJECT_SOURCE_DIR}/../llpc/tool/vfx")
```

Comment:
Why this new "if"? Can't you just unconditionally set it FORCE?

Also set the one below FORCE, otherwise an incremental build will break switching between old and new vfx locations.

Reply:
Hi @trenouf , is the new change as expected?

Reply:
Hi Xue. Sorry, not quite what I expected. Please see comments on the file.

Reply:
Fixed. Thank you for the detailed comments.